Jenkinsfile

```diff
@@ -10,13 +10,11 @@ properties([
     ]
 ])
 
-def SDK_URL = "master/24.32.0/1.xs8/76/"
-
 def builder = null
 def globals = globals()
 
 try {
-    builder = new Build(globals, SDK_URL)
+    builder = new Build(globals)
```

Comment:
After remove SDK_URL, how could we test the dev SDK? I remember we add this parameter for build samples of dev SDK.

Reply:
The SDK URL will be specified in the internal pipeline. In this repo all we need to do is specify the pipeline branch to test.